examples/counter_in/cnt_dig_buff_cont_ext_clk.py

```diff
@@ -0,0 +1,35 @@
+﻿"""Example of counter input edge count operation.
+
+This example demonstrates how to count buffered digital events
+on a Counter Input Channel. The Initial Count, Count Direction,
+Edge, and Sample Clock Source are all configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, CountDirection, Edge
+
+with nidaqmx.Task() as task:
+    task.ci_channels.add_ci_count_edges_chan(
+        "Dev1/ctr0",
+        edge=Edge.RISING,
+        initial_count=0,
+        count_direction=CountDirection.COUNT_UP,
+    )
```

Comment:
We should show people how to change the terminal that is being counted on.

```suggestion
    channel = task.ci_channels.add_ci_count_edges_chan(
        "Dev1/ctr0",
        edge=Edge.RISING,
        initial_count=0,
        count_direction=CountDirection.COUNT_UP,
    )
    channel.ci_count_edges_term = "Dev1/PFI8"
```